static/app/components/events/interfaces/request/graphQlRequestBody.tsx

```diff
@@ -41,6 +47,32 @@ function getErrorLineNumbers(errors: GraphQlErrors): number[] {
   );
 }
 
+function ErrorsAlert({errors}: {errors: GraphQlErrors}) {
+  if (isEmpty(errors)) {
+    return null;
+  }
+
+  return (
+    <StyledAlert
+      type="error"
+      showIcon
+      expand={
+        <List symbol="bullet">
+          {errors.map((error, i) => (
+            <li key={i}>{error.message}</li>
```

Comment:
Based on the spec, `message` is a must https://spec.graphql.org/October2021/#sec-Errors.Error-result-format
But It's hard to tell if every GraphQL impl. follows this rule, can we guard if `message` isn't actually null/empty? so it's not adding the item if there's nothing to be shown.

Reply:
Guarded against this in [`00eaf3b` (#51846)](https://github.com/getsentry/sentry/pull/51846/commits/00eaf3b56c495fa100281c8f024af7c073b21751)